extensions/smallrye-metrics/runtime/src/main/java/io/quarkus/smallrye/metrics/runtime/SmallRyeMetricsRecorder.java

```diff
@@ -209,6 +215,94 @@ private void operatingSystemMetrics(MetricRegistry registry, List<String> names)
                 .build();
         registry.register(meta, new LambdaGauge(() -> operatingSystemMXBean.getAvailableProcessors()));
         names.add(CPU_AVAILABLE_PROCESSORS);
+
+        // some metrics are only available in jdk internal class 'com.sun.management.OperatingSystemMXBean': cast to it.
+        // com.sun.management.OperatingSystemMXBean is not available in SubstratVM
+        if (!ImageInfo.inImageCode()) {
+            try {
+                // the cast will fail for some JVM not derived from HotSpot (J9 for example)
+                com.sun.management.OperatingSystemMXBean internalOperatingSystemMXBean = (com.sun.management.OperatingSystemMXBean) operatingSystemMXBean;
+                meta = Metadata.builder()
+                        .withName(PROCESS_CPU_LOAD)
+                        .withType(MetricType.GAUGE)
```

Comment:
```suggestion
                        .withType(MetricType.GAUGE)
                        .withUnit(MetricUnits.PERCENT)
```

The spec says that the unit of this metric is percent, it affects the name of the exported metric in OpenMetrics export

Reply:
done